### PR TITLE
Mark as compatible with React 15

### PR DESCRIPTION
**Pull request by ronny**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,15 +51,15 @@
...
+    "react-addons-test-utils": "^0.14.8 || ^15.0.0",
     "react-dom": "^0.14",
     "require-dir": "^0.3.0",
     "webpack": "^1.12.12",
     "webpack-dev-server": "^1.14.1"
   },
   "peerDependencies": {
-    "react": "^0.14",
-    "react-dom": "^0.14"
+    "react": "^0.14.0 || ^15.0.0-0",
```
Comment (FrancescoCioria) [inline package.json]: can please remove the `-0` at the end?

File: package.json [hunk 2]
```diff
@@ -51,15 +51,15 @@
...
+    "react-dom": "^0.14.0 || ^15.0.0-0"
```
Comment (FrancescoCioria) [inline package.json]: here too :)

**Comment by FrancescoCioria** [comment]
@ronny thank you for your PR! I left you two small comments :)

**Comment by ronny** [comment]
@FrancescoCioria Done! Thank you.

**Comment by FrancescoCioria** [comment]
great!

**Pull request merged**